dev/com.ibm.ws.security.social/resources/com/ibm/ws/security/social/resources/SocialMessages.nlsprops

```diff
@@ -42,23 +42,47 @@ SOCIAL_LOGIN_CONFIG_ATTRIBUTE_EMPTY.explanation=The value for this configuration
-OPENSHIFT_USER_API_BAD_STATUS.explanation=The OpenShift user API did not return the expected status code.
-OPENSHIFT_USER_API_BAD_STATUS.useraction=Check the API response that is in the error message for more information. Verify that the request to the API contains all of the required information. Ensure that the OpenShift service account token was created in the correct OpenShift project and has the correct permissions.
+KUBERNETES_USER_API_BAD_STATUS=CWWKS5373E: The Kubernetes user API returned an unexpected [{0}] response code. Verify that the request to the API contains all of the required information and that the Kubernetes service account token was created with the correct permissions. The response from the API is [{1}].
+KUBERNETES_USER_API_BAD_STATUS.explanation=The Kubernetes user API did not return the expected status code.
+KUBERNETES_USER_API_BAD_STATUS.useraction=Check the API response that is in the error message for more information. Verify that the request to the API contains all of the required information. Ensure that the Kubernetes service account token was created the correct permissions.
 
 # 0=JSON key string, 1=User API response string
-OPENSHIFT_USER_API_RESPONSE_MISSING_KEY=CWWKS5374E: The OpenShift user API response is missing an expected key: [{0}]. The full response is [{1}]
-OPENSHIFT_USER_API_RESPONSE_MISSING_KEY.explanation=The key that is specified in the message is expected to be within the API response. The key might be missing, or it might be in an unexpected location.
-OPENSHIFT_USER_API_RESPONSE_MISSING_KEY.useraction=Check the API response to determine whether the key is missing from the response.
+KUBERNETES_USER_API_RESPONSE_MISSING_KEY=CWWKS5374E: The Kubernetes user API response is missing an expected key: [{0}]. The full response is [{1}]
+KUBERNETES_USER_API_RESPONSE_MISSING_KEY.explanation=The key that is specified in the message is expected to be within the API response. The key might be missing, or it might be in an unexpected location.
+KUBERNETES_USER_API_RESPONSE_MISSING_KEY.useraction=Check the API response to determine whether the key is missing from the response.
+
+# 0=Social login configuration ID
+ACCESS_TOKEN_MISSING_FROM_HEADERS=CWWKS5375E: The social login configuration [{0}] specifies that an access token is required in the request, but an access token is not present.
+ACCESS_TOKEN_MISSING_FROM_HEADERS.explanation=The social login configuration that is specified in the message requires that all incoming requests must contain an access token.
+ACCESS_TOKEN_MISSING_FROM_HEADERS.useraction=If an access token header name is configured in the social login configuration, verify that the header is included in the request. Otherwise, verify that an Authorization header is included in the request. Ensure that the header value conforms to the OAuth bearer token specification. Verify that the OAuth provider is configured to pass an access token.
```

Comment:
It looks like the capitalization change for "Authorization header" is the only suggested change here? I'd like to keep "Authorization" capitalized because it's the name of a well-defined header as opposed to some generic authorization header.